src/views/partials/events/CreateEventModal.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <button
+    class="btn flex-none mt-1 w-24 h-8 bg-emerald-500 hover:bg-emerald-800 text-white font-bold py-1 px-2 ml-2 rounded"
+    type="button"
+    @click="toggleModal()"
+  >
+    <i class="fas fa-plus mr-2"></i>Create
+  </button>
+
+  <div v-show="isOpen" class="absolute inset-0 flex items-center justify-center bg-black-deep bg-opacity-70">
+    <div class="relative bg-white rounded-lg shadow dark:bg-black-light w-1/2">
+      <div class="flex justify-between items-start p-4 rounded-t border-b dark:border-gray-600">
+        <h3 class="text-xl font-semibold text-gray-900 dark:text-white m-0">Create Event Position</h3>
+        <button
+          type="button"
+          class="text-gray-400 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm p-1.5 ml-auto inline-flex items-center dark:hover:bg-gray-600 dark:hover:text-white"
+          @click="toggleModal()"
+        >
+          <svg
```

Comment:
Might be cleaner to use font awesome as it's already loaded?

Reply:
Fixed in [b401cbc]